docs-client/src/containers/HomePage/index.tsx

```diff
@@ -14,11 +14,54 @@
  * under the License.
  */
 
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
 import Typography from '@material-ui/core/Typography';
 import React from 'react';
+import Section from '../../components/Section';
+import {
+  convertLongToUTCDate,
+  extractSimpleArtifactVersion,
+  Versions,
+} from '../../lib/versions';
 
-export default class HomePage extends React.PureComponent {
+interface OwnProps {
+  versions?: Versions;
+}
+
+type Props = OwnProps;
+
+export default class HomePage extends React.PureComponent<Props> {
   public render() {
+    const { versions } = this.props;
+    const data = versions;
+    const tableRows: any = [];
+
+    if (data) {
+      data.getVersions().forEach((version) => {
+        tableRows.push(
+          <TableRow>
+            <TableCell>{version.artifactId}</TableCell>
+            <TableCell>
+              {extractSimpleArtifactVersion(version.artifactVersion)}
+              {data.getArmeriaArtifactVersion() !==
+              extractSimpleArtifactVersion(version.artifactVersion)
+                ? '(!)'
+                : ''}
+            </TableCell>
+            <TableCell>
+              {convertLongToUTCDate(version.commitTimeMillis)}
+            </TableCell>
+            <TableCell>{version.shortCommitHash}</TableCell>
```

Comment:
Are you going to add a tooltip for this? It's up to you - we can add it later.

Reply:
okay. I can develop tooltip.